packages/eslint-config-office-addins/package.json

```diff
@@ -4,9 +4,12 @@
   "description": "Provides an eslint configuration for Office Add-ins.",
   "main": "./lib/main.js",
   "scripts": {
-    "tsc": "tsc -p tsconfig.json",
     "build": "rimraf lib && concurrently \"npm run tsc\"",
-    "test": "echo \"No tests\""
+    "lint": "office-addin-lint check",
+    "lint:fix": "office-addin-lint fix",
+    "prettier": "office-addin-lint prettier",
+    "test": "echo \"No tests\"",
+    "tsc": "tsc -p tsconfig.json"
```

Comment:
Same comment about "tsc". Please go through all the packages and don't add tsc to them.